frontend/packages/console-shared/src/components/status/generic-status.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import {
+  HourglassStartIcon,
+  HourglassHalfIcon,
+  SyncIcon,
+  BanIcon,
+  WarningTriangleIcon,
+  UnknownIcon,
+} from '@patternfly/react-icons';
+import {
+  StatusIconAndText,
+  ProgressStatus,
+  ErrorStatus,
+  SuccessStatus,
+  InfoStatus,
+} from './status';
+
+export type GenericStatusProps = {
+  status?: string;
+};
+
+export const GenericStatus: React.FC<{ status?: string }> = ({ status, children }) => {
```

Comment:
I think `GenericStatus` is a bad name.
`React.FC<{ status?: string }>` should reference the prop type